openfe/tests/setup/test_molecule.py

```diff
@@ -1,5 +1,12 @@
+import openff.toolkit.topology
 import pytest
 
+try:
+    from openeye import oechem
+except ImportError:
+    HAS_OECHEM = False
+else:
+    HAS_OECHEM = oechem.OEChemIsLicensed()
```

Comment:
Looking at the coverage, it looks like the import is working but the license check is failing.  Is this something @mikemhenry that you can help with?

Reply:
:+1:

Reply:
I think this part needs to be moved up

```
diff --git a/.github/workflows/ci.yaml b/.github/workflows/ci.yaml
index 14bc09c..b279758 100644
--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -45,6 +45,13 @@ jobs:
             environment-file: environment.yml
             activate-environment: openfe
 
+      - name: Decrypt OpenEye license
+        env:
+          OE_LICENSE_TEXT: ${{ secrets.OE_LICENSE }}
+        run: |
+          echo "${OE_LICENSE_TEXT}" > ${OE_LICENSE}
+          python -c "import openeye; assert openeye.oechem.OEChemIsLicensed(), 'OpenEye license checks failed!'"
+
       - name: "Install"
         run: python setup.py develop --no-deps
 
@@ -61,13 +68,6 @@ jobs:
           mamba info -a
           mamba list
 
-      - name: Decrypt OpenEye license
-        env:
-          OE_LICENSE_TEXT: ${{ secrets.OE_LICENSE }}
-        run: |
-          echo "${OE_LICENSE_TEXT}" > ${OE_LICENSE}
-          python -c "import openeye; assert openeye.oechem.OEChemIsLicensed(), 'OpenEye license checks failed!'"
-
       - name: "Run tests"
         run: |
           pytest -n 2 -v --cov=openfe --cov-report=xml openfe/tests/
```

Reply:
git has gotten a bit weird, I can commit this fix in a bit, but need to fix my branch which has some merge conflicts with upstream

Reply:
Cool looks like you got it it with [69dbfd8](https://github.com/OpenFreeEnergy/openfe/pull/58/commits/69dbfd862ae6429d8c8dbe0a611904916a5c2720)